pkgs/applications/graphics/gimp/plugins/default.nix

```diff
@@ -387,7 +326,6 @@ lib.makeScope pkgs.newScope (
 
         license = lib.licenses.gpl3Plus;
         maintainers = [ ];
-        platforms = lib.platforms.gnu ++ lib.platforms.linux;
```

Comment:
I wonder if we still should have `meta.platforms`: https://nixos.org/manual/nixpkgs/stable/#var-meta-platforms
Though I see that others do not have that, and Hydra will still build it thanks to `gimp-with-plugins` depending on it.

Reply:
On the other hand, if support for one of platforms supported by  `gimp-with-plugins` was missing from a plugin, `gimp-with-plugins` would be broken on that platform. And plug-in definitely cannot support more platforms since it depends on GIMP itself. So `meta.platforms` is actually pointless.

Reply:
That documentation has been out of date since 2016; see https://github.com/NixOS/nixpkgs/pull/19990 and https://github.com/NixOS/nixpkgs/pull/351919. (It should probably be updated. I didn’t know it said that until right now.)